### ISSUE TITLE
Failed pytest, test_uniprot_functionality

### ISSUE THREAD

**Issue by RossLabCSU** [idptools/metapredict#2]
Probably a question more than an issue...
When I run pytest -v, one check fails. Below is the error message.
![Metapredict_test_screenshot](https://user-images.githubusercontent.com/49657843/142781341-bb26ae31-0406-49dc-8b7f-28ba6fec2965.png)

I'm running from a Windows machine with conda: do you think this is caused by the known clash with conda-installed numpy, or is this simply a rounding difference? When I run the test manually within my own python script on a manually downloaded p53 sequence from Uniprot and use np.sum() for the test, I also get the same value that occurred during the pytest (173.04537763974952). However, when I run the same test with Python's built-in sum() function, I get 173.04537763974955 (maybe this also is a known behavior difference between np.sum() and built-in sum()).

More importantly, given the miniscule difference between the values compared by pytest, is it safe to say that identical regions are still being classified as disordered?

Thanks!
-Sean

**Comment by alexholehouse** [comment]
Sean - thanks so much! 

So we haven't tested extensively in a Windows environment and honestly yes, this is probably just a rounding error (we should really be using numpys .isclose() function here rather than hardcoding precision-dependent vals in). 

I'll take a look over the weekend and will probably push an update that changes how the comparison is being done to use isclose() - once this is done I'll post here and if the tests pass in your env. that would be superhelpful to know!

Thanks again - more soon!

**Comment by alexholehouse** [comment]
Just to follow up, we think in our most recent update we have fixed this (along with a bunch of other things). If this remains an issue let me know, but we just pushed a MAJOR update which has some big changes so there may a bug or two remaining... Closing this now, but please re-open if this is not resolved OR open a new issue when necessary!
